src/openscm_units/_unit_registry.py

```diff
@@ -232,7 +232,8 @@ def add_standards(self) -> None:
 
         self._add_gases({x: x for x in MIXTURES})
 
-        self.define("a = 1 * year = annum = yr")
+        self.define("yr = 1 * year")
+        self.define("a = 1 * year = annum")
```

Comment:
Very subtle change. It wasn't obvious to me straight away that 'everything on the right is an aliases for the first thing on the left'